### PR TITLE
aw - several fixes

### PR DESCRIPTION
**Pull request by ghost**
- cscrootcerts.crt plural
- ubuntugis ppa
- ~gis libs use ssl~
- ogr settings
- ssl test
- - #!/shebangs
- [x] tested

### CODE REVIEW

File: src/rootcert.sh
Comment (pantelisindefra) [inline src/rootcert.sh:1]: Please keep the comment # Rest is taken from https://kb.databricks.com/python/import-custom-ca-cert at the place it was.

**Review by pantelisindefra — COMMENTED** [review]
Looks good, you test it, I will approve it.

**Comment by ghost** [comment]
SSL fails for GIS+GDAL init scripts - I am unsure why, but it works without SSL, so leaving for the time being.


Successfully tested optional scripts with this `DASH.sh` in PRD'2406 on Notebook cluster.
```sh
#!/bin/bash
set -ex
echo $DB_CLUSTER_NAME

URL=https://raw.githubusercontent.com/Defra-Data-Science-Centre-of-Excellence/init-scripts/aw-fixes/src
wget $URL/rootcert.sh -O- | bash -ex
wget $URL/update.sh -O- | bash -ex
wget $URL/base_libs.sh -O- | bash -ex
wget $URL/r_installs.R -O- | Rscript -
wget $URL/gis_libs.sh -O- | bash -ex
wget $URL/gdal.sh -qO- | bash -ex
```

**Comment by ghost** [comment]
@pantelisindefra 
I didn't get SSL to work on init scripts, but I fix a few small errors.

**Review by Fraser-Buchanan — APPROVED** [review]
Looks good. We probably need to chat about the dbfs clean script at some point.

**Pull request merged**